tests/testthat/test-derive_extreme_records.R

```diff
@@ -1,22 +1,53 @@
+adsl <- tibble::tribble(
+  ~USUBJID, ~DTHDT,
+  "1",      ymd("2022-05-13"),
+  "2",      ymd(""),
+  "3",      ymd("")
+) %>%
+  mutate(STUDYID = "XX1234")
+
+adrs <- tibble::tribble(
+  ~USUBJID, ~ADTC,        ~AVALC, ~PARAMCD,
```

Comment:
I think we have been trying to avoid having data outside the test_that script as they can spill into other tests - maybe have as a fixture?  https://r-pkgs.org/testing-advanced.html#sec-testing-advanced-concrete-fixture

Reply:
Good point! In this case the dataset created at the top were even overwritten after some tests. We should definitely avoid this.

I have moved the test data into the test. Using fixtures in this case does not seem appropriate to me. I think the tests would be hard to read and maintain if we create the test data outside the test file.